### ISSUE TITLE
How to get the current day displayed in the view ?

### ISSUE THREAD

**Issue by DaConceicao-Nelson** [alamkanak/Android-Week-View#17]
Hi.
First, thank you for the implementation of this library. It's a good job !

I would like to use it to display and add events. To do this, I need to know the day shown by the view to add an new event. But at this moment, I can't find a callback method in the library that lets me know the day displayed. Is this implemented or not?

I tried to make my own implementation of the following way:

```
private void drawHeaderRowAndEvents(Canvas canvas) {
...
// Iterate through each day.
        for (int dayNumber = leftDaysWithGaps + 1;
             dayNumber <= leftDaysWithGaps + mNumberOfVisibleDays + 1;
             dayNumber++) {

            // Check if the day is today.
            day = (Calendar) mToday.clone();
            day.add(Calendar.DATE, dayNumber - 1);
            boolean sameDay = isSameDay(day, mToday);

            // Return the current date displayed
            if (dayNumber == leftDaysWithGaps + 1 && !mFetchedDate.equals(day)) {
                mFetchedDate = (Calendar) day.clone();
                // Callback method like "onMonthChange()" to know the current day displayed
                mDayChangeListener.onDayChange(mFetchedDate);
            }
...
}
```

In short, my question is : The library implements a way to know the day displayed ? If so, can you explain me how ? If not, my solution is it right ?

Thank's for your help !
